Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -379,7 +379,7 @@ To add a new service that should only be exposed to nodes with label ``service.c
   kind: Service
   metadata:
     name: example-service
-    labels:
+    annotations:
```

Comment:
After careful consideration, I've come to the conclusion that this major doc change is acceptable :slightly_smiling_face:. Thanks!